tools/spectral/ipa/rulesets/IPA-105.yaml

```
@@ -29,3 +30,23 @@ rules:
     then:
       field: '@key'
       function: 'eachResourceHasListMethod'
+  xgen-IPA-105-list-method-response-is-get-method-response:
+    description: >-
+      The response body of the List method should consist of the same resource object returned by the Get method. http://go/ipa/105
```

Link to IPA is no longer needed as it is already part of the docs and in failure we do reference to the docs in open source repo
```suggestion
      The response body of the List method should consist of the same resource object returned by the Get method. 
```

---

Meta comment/ Not this PR/Project:

In retrospect we could use public shortener to link to this repo to be kind for non MongoDB use cases. 
Technically you could make blog post covering new mongodb validations and then go links will be the only blocker. All the info will be availabile upstream.
This can be addressed once we get our plans around IPA publication done.